tests/sb-util.test.ts

```diff
@@ -0,0 +1,33 @@
+import { initialize, ScratchProject } from '../src/sb-util';
```

Comment:
`initialize` is a little too generic I think ;)

Reply:
I agree. What do you think is a better name?

Reply:
Per conversation with @rwaldron, I'm going to make a ScratchLoader class or something like this that is solely responsible for init

Reply:
Broke this out into 3 functions.